<!DOCTYPE HTML>
<html lang="en">

<head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

  <title>Reyhane Ghorbani Nia</title>

  <meta name="author" content="Jon Barron">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <link rel="stylesheet" type="text/css" href="stylesheet.css">
  <link rel="icon" type="image/jpg" href="Data/profile_face.png">
</head>


<body>
  <table
    style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
    <tbody>
      <tr style="padding:0px">
        <td style="padding:0px">
          <table
            style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
            <tbody>
              <tr style="padding:0px">
                <td style="padding:2.5%;width:63%;vertical-align:middle">
                  <p style="text-align:center">
                    <name>Reyhane Ghorbani Nia</name>
                  </p>
                  <p>

                    I am a research assistant at the University of California, Riverside (UCR), where we leverage computational algorithms to design compact Cas9-sgRNA libraries for CRISPR genome editing across fungal species. This work facilitates the production of novel compounds, unlocking fungi's potential for diverse applications, including sustainable biofuels, eco-friendly biomaterials, life-saving pharmaceuticals, and innovative food products.

                    <p>
                    I earned my MS in Chemical and Environmental Engineering from University of California, Riverside under supervision of Professor <a href="https://profiles.ucr.edu/app/home/profile/wheeldon"><b>Ian Wheeldon</b></a> , and prior to that, I completed my bachelor’s degree in chemical engineering at the University of Tehran. Throughout my research journey, I had the privilege of collaborating closely with Professor Stefano Lonardi from the University of California, Riverside, and Professor Abbas Ali Khodadadi from the University of Tehran, whose mentorship greatly enriched my work. 
                    </p>


                  </p>
                  <p style="text-align:center">
                    <a href="mailto:rghor005@ucr.edu">Email</a> &nbsp/&nbsp
                    <a href="https://scholar.google.com/citations?user=M4qM9-YAAAAJ&hl=en">Google Scholar</a>
                    &nbsp/&nbsp
                    <a href="https://github.com/reyghorbani">GitHub</a> &nbsp/&nbsp
                    <!-- <a href="">Twitter</a> &nbsp/&nbsp -->
                    <a href="https://www.linkedin.com/in/reyhane-g-467b8b17b/">LinkedIn</a> &nbsp/&nbsp
                    <a href="Data/CV_Reyhane Ghorbani Nia_2024.pdf">CV</a>
                  </p>
                </td>
                <td style="padding:2.5%;width:40%;max-width:40%">
                  <a href="Data/profile_circle.png"><img style="width:100%;max-width:100%" alt="profile photo"
                      src="Data/profile_circle.png" class="hoverZoomLink"></a>
                </td>
              </tr>
          </table>



          <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
            <tr>
              <td width="100%" valign="middle">
                <heading>News</heading>
                <!-- <br /> -->


                <ul>
                  <li><b>November 2024:</b> Thrilled to announce that I was honored with the <b>Best Teaching Assistant Award</b> from the University of California, Riverside! It's an incredible recognition of my dedication to teaching and mentoring students. </li>
                </ul>


                <ul>
                  <li><b>October 2024:</b> Excited to share that our recent work on, “ALLEGRO: Efficient Design of Cas9-gRNA Libraries of Minimal Size Targeting Thousands of Genomes,” was selected for a poster presentation at the prestigious <b>7th International Conference on CRISPR Technologies</b>. </li>
                </ul>


                <ul>
                  <li><b>September 2024:</b> Grateful to have received the <b>AIChE Travel Grant</b> to participate in the <b>7th International Conference on CRISPR Technologies</b>, supporting the dissemination of our cutting-edge research. </li>
                </ul>

                <ul>
                  <li><b>September 2024:</b> Officially earned my <b>Master's degree in Chemical and Environmental Engineering</b> from the University of California, Riverside. A proud milestone in my academic journey! </li>
                </ul>

                <ul>
                  <li><b>August 2024:</b> Achieved an <b>online certification in Python for Genomic Data Science</b> from Johns Hopkins University. Honing my skills to stay at the forefront of computational biology! </li>
                </ul>

                <ul>
                  <li><b>January 2023:</b> Began an exciting new role as a  <b>Research Assistant in Wheeldon's Lab</b> at the University of California, Riverside, delving into groundbreaking research in CRISPR genome editing and metabolic engineering. </li>
                </ul>

                <ul>
                  <li><b>October 2022:</b> Honored to have been awarded the <b>Dean's Distinguished Fellowship</b> from the Department of Chemical and Environmental Engineering at the University of California, Riverside. This recognition fuels my passion for impactful research and innovation. </li>
                </ul>


              </td>
            </tr>
          </table>





          <table
            style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
            <tbody>
              <tr>
                <td style="padding:20px;width:100%;vertical-align:middle">
                  <heading>Research</heading>
                  <p>

                    My primary research centers on computational biology and CRISPR-Cas9 knockout screenings. In the past, I have explored diverse areas including drug delivery, metal-organic frameworks (MOFs), and molecular dynamics simulations.

                  </p>
                </td>
              </tr>
          </table>
          <table
            style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
            <tbody>
              

              <tr onmouseout="allergo_stop()" onmouseover="allergo_start()">
                <td style="padding:20px;width:25%;vertical-align:middle">
                  <div class="one">
                    <div class="two" id='allergo_image'>
                      <img src='Data/allergo_before.png' width="160">
                    </div>
                    <img src='Data/allergo_after.png' width="160">
                  </div>
                  <script type="text/javascript">
                    function allergo_start() {
                      document.getElementById('allergo_image').style.opacity = "0";
                    }

                    function allergo_stop() {
                      document.getElementById('allergo_image').style.opacity = "1";
                    }
                    allergo_stop()
                  </script>
                </td>
                <td style="padding:20px;width:75%;vertical-align:middle">
                  <a href="">
                    <papertitle> ALLEGRO: Efficient Design of Cas9-gRNA Libraries of Minimal Size Targeting Thousands of Genomes
                    </papertitle>
                  </a>
                  <br>

                  Amirsadra Mohseni*, <strong>Reyhane Ghorbani Nia*</strong>, Aida Tafrishi, Xinzhan Liu, Jason Stajich, Stefano Lonardi, Ian Wheeldon

                  <!-- <br> -->
                  <!-- <em>ICML Workshop on the Next Generation of AI Safety</em>, 2024 -->
                  <!-- <font color="red"><strong>(Oral Presentation)</strong></font> -->
                  <!-- <br> -->
                  <!-- <a href="">PDF</a> /
                  <a href="">arXiv</a> /
                  <a href="">code</a> -->
                  <br>
                  <p></p>
                  <p>
                    In this work, we introduce ALLEGRO, a computational method leveraging combinatorial optimization to design minimal gRNA libraries for thousands of fungal species. ALLEGRO efficiently identifies orthologous genes, extracts candidate gRNAs, predicts their activity scores, and employs linear programming to generate the smallest high-activity gRNA libraries within minutes, enabling precise and scalable genome editing.
                  </p>
                </td>
              </tr>

              
              <tr onmouseout="sup_stop()" onmouseover="sup_start()">
                <td style="padding:20px;width:25%;vertical-align:middle">
                  <div class="one">
                    <div class="two" id='sup_image'>
                      <img src='Data/sup_before.png' width="160">
                    </div>
                    <img src='Data/sup_after.png' width="160">
                  </div>
                  <script type="text/javascript">
                    function sup_start() {
                      document.getElementById('sup_image').style.opacity = "0";
                    }

                    function sup_stop() {
                      document.getElementById('sup_image').style.opacity = "1";
                    }
                    sup_stop()
                  </script>
                </td>
                <td style="padding:20px;width:75%;vertical-align:middle">
                  <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ceat.202200588">
                    <papertitle> Supercritical Methanol and Ethanol Solubility Estimation by Using Molecular Dynamics Simulation
                    </papertitle>
                  </a>
                  <br>

                  Hojatollah Moradi, Dr. Hedayat Azizpour, Dr. Parissa Khadiv-Parsi, <strong>Reyhane Ghorbani Nia</strong> 
                  <br>
                  <em>Journal: Chemical Engineering Technology, Wiley</em>, 2023
                  <br>
                  <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/ceat.202200588">PDF</a>
                  <!-- <a href="">arXiv</a> -->
                  <!-- <a href="">code</a> -->
                  <br>
                  <p></p>
                  <p>
                    In this study, we explored the solubility parameters of supercritical methanol and ethanol through molecular dynamics simulations across varying temperatures and pressures. The predicted solubility parameters closely align with theoretical data, validating our approach. Electrostatic interactions were modeled using the Ewald summation method, while van der Waals interactions were handled with the atom-based summation method. Additionally, our findings demonstrate that increasing density linearly improves solubility.</p>
                </td>
              </tr>



              <tr onmouseout="pred_stop()" onmouseover="pred_start()">
                <td style="padding:20px;width:25%;vertical-align:middle">
                  <div class="one">
                    <div class="two" id='pred_image'>
                      <img src='Data/pred_before.png' width="160">
                    </div>
                    <img src='Data/pred_after.png' width="160">
                  </div>
                  <script type="text/javascript">
                    function pred_start() {
                      document.getElementById('pred_image').style.opacity = "0";
                    }

                    function pred_stop() {
                      document.getElementById('pred_image').style.opacity = "1";
                    }
                    pred_stop()
                  </script>
                </td>
                <td style="padding:20px;width:75%;vertical-align:middle">
                  <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ceat.202200629">
                    <papertitle> Prediction of Water-Methanol Mixture Properties by Molecular Dynamics Simulation
                    </papertitle>
                  </a>
                  <br>
                  Hojatollah Moradi, Dr. Hedayat Azizpour, Dr. Kamran Keynejad, <strong>Reyhane Ghorbani Nia</strong>, Dr. Amin Esmaeili
                  <br>
                  <em>Journal: Chemical Engineering Technology, Wiley</em>, 2023
                  <br>
                  <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/ceat.202200629">PDF</a>
                  <!-- <a href="https://arxiv.org/abs/2310.19909">arXiv</a> /
                  <a href="https://github.com/hsouri/Battle-of-the-Backbones">code</a> -->
                  <br>
                  <p></p>
                  <p>
                    In this work, we utilize molecular dynamics (MD) simulation and empirical data to predict the density of water-methanol mixtures at 293.15–303.15K and atmospheric pressure. Our approach employs the Ewald summation method for electrostatic interactions and the atom-based method for van der Waals interactions. We further apply the Jouyban-Acree (J-A) model and density correlation methods for two-component mixtures to analyze MD simulation results across various temperatures and methanol mole fractions.
                  </p>
                </td>
              </tr>




              <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
                <tr>
                  <td width="100%" valign="middle">
                    <heading>Research Experience</heading>
                    <!-- <br /> -->

                    <ul>
                      <li>Research Assistant, University of California, Riverside, January 2023 - Present</li>
                    </ul>
                    <ul>
                      <li>Research Assistant, University of Tehran, September 2021 - December 2022</li>
                    </ul>
                  </td>
                </tr>
              </table>


              <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
                <tr>
                  <td width="100%" valign="middle">
                    <heading>Teaching Experience</heading>
                    <!-- <br /> -->
                    <ul>
                      <li>Teaching Assistant, Machine Intelligence, University of California, Riverside, Spring 2023, Fall 2024
                      </li>
                    </ul>

                    <ul>
                    <li>Teaching Assistant, University of Tehran, Fall 2019 - Spring 2022 </li>
                    </ul>
                  </td>
                </tr>
              </table>


              <table
                style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
                <tbody>
                  <tr>
                    <td style="padding:0px">
                      <br>
                      <p style="text-align:center;font-size:small;">
                        <a href="https://jonbarron.info/">Source code</a>
                      </p>
                      <p style="text-align:center;font-size:small;">
                        Last update: December, 2024</p>

                    </td>
                  </tr>
                </tbody>
              </table>
        </td>
      </tr>
  </table>
</body>

</html>